3-networks-dual-svpc/envs/shared/remote.tf

```diff
@@ -17,12 +17,10 @@
 locals {
   env                       = "common"
   environment_code          = "c"
-  dns_bgp_asn_number        = var.enable_partner_interconnect ? "16550" : var.bgp_asn_dns
+  dns_bgp_asn_number        = var.bgp_asn_dns
```

Comment:
where is this local used? there is no interconnect anymore

Reply:
Removed.